assets/src/dashboard/app/views/editorSettings/videoCache/stories/index.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs';
+import { FlagsProvider } from 'flagged';
+
+/**
+ * Internal dependencies
+ */
+import VideoCacheSettings from '..';
+
+export default {
+  title: 'Dashboard/Views/EditorSettings/VideoCache',
+  component: VideoCacheSettings,
+};
+
+export const _default = () => {
+  return (
+    <FlagsProvider features={{ videoCache: true }}>
+      <VideoCacheSettings
+        isEnabled={boolean('isEnabled', true)}
+        updateSettings={action('updateSettings fired')}
+      />
+    </FlagsProvider>
+  );
+};
```

Comment:
Where is this export/file used?

Reply:
This a storybook component. It's imported by storybook.

Our storybook lives at https://google.github.io/web-stories-wp/storybook/